### PR TITLE
add a post-mortem for prow outage 2019-02-08

### PR DESCRIPTION
**Pull request by krzyzacy**
Open as a draft in case I forgot, and still trying to understand the root cause and why this is not deterministic (maybe @stevekuznetsov can explain a bit better)
/assign

### CODE REVIEW

**Comment by stevekuznetsov** [comment]
Yeah actually re-reading the broken code I don't fully understand why it was not deterministic

**Comment by spiffxp** [comment]
Does this need more refinement or shall we merge it as is?

**Comment by krzyzacy** [comment]
@spiffxp seems we are unsure why the behavior is not deterministic... I'll just add a note

File: docs/post-mortems/2019-02-08.md [hunk 1]
```diff
@@ -0,0 +1,382 @@
...
+
+> stevekuznetsov [5:06 PM]
+(why is the kubeconfig for the service cluster even in your clusters yaml given to plank to schedule pods????)
+
+> stevekuznetsov [5:06 PM]
+Also I need to jump but if you confirm the SHA bounds that @Katharine posted ... there is no suspicious code change in that deployment AFAICT
+
+```
+$ git log --no-merges --pretty='%cd | %h | %<(20)%an | %s' --date=short b01b288..c35c7e3 -- prow/ ':!prow/config.yaml' ':!prow/plugins.yaml' ':!prow/cluster'
+2019-02-07 | 73ab4a815 | Kyle Weaver          | Deck: add missing org to Github links
```
Comment (spiffxp) [inline docs/post-mortems/2019-02-08.md]:
```suggestion
2019-02-07 | 73ab4a815 | Kyle Weaver          | Deck: add missing org to GitHub links
```

**Comment by spiffxp** [comment]
I was unable to add my suggestions to this PR
Screen Shot 2019-04-19 at 4 10 53 PM

According to https://help.github.com/en/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork this seems to be a setting the PR author has to enable, and I couldn't find anything in repo settings to enable by default (which makes sense I guess as an anti-abuse measure)

/close
So, closing this in favor of https://github.com/kubernetes/test-infra/pull/12285 which has the typo fix

**Pull request closed without merging**